README.md

```diff
@@ -80,6 +80,10 @@ Review, Event, People, Product, Recipe, Software Application, Video, Articles et
 
 ## Changelog ##
 
+### 1.6.10 ###
+- Fixed: Added the uploadDate field to the $video string in the correct ISO 8601 format with timezone.
```

Comment:
No need to mention such details $video in the changelog, use below string

Corrected the uploadDate format as per ISO 8601 standards with timezone